src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/helper/JwtAuthHelper.kt

```diff
@@ -30,7 +29,7 @@ class JwtAuthHelper {
   fun jwtDecoder(): JwtDecoder = NimbusJwtDecoder.withPublicKey(keyPair.public as RSAPublicKey).build()
 
   fun setAuthorisation(
-    user: String = "visit-scheduler-client",
+    user: String = "hmpps-manage-prison-visits-orchestration-client",
```

Comment:
hmm this must have been wrong for a while

Reply:
yes, I think I copied it from visit-scheduler but didn't update the client.